CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### 8.0.2
+- Add `Aws::SQS::Errors::AccessDenied` to the list of errors that Sqewer must release the singleton SQS client
```

Comment:
Just saw this one. Hmm, I think this would be raised also if the queue does not have the correct permissions, in which case it would trigger quite a lot of credential checks, given they won't be cached at all 🤔 

I'm not sure of the ideal solution to this, but as I mentioned in the other issue, it doesn't necessarily feel like something that needs addressing on the client level 🤔

Reply:
yes, I agree, but until we find the perfect/correct solution, I think it is the option we have 🤷‍♂️

Reply:
Yeah, let's just not forget to look into it 🙌